src/taskManager/include/taskManager/AbstractTask.h

```diff
@@ -0,0 +1,26 @@
+#include <algorithm>
```

Comment:
The include should be after the `ifndef`

Reply:
Fixed